src/lib.rs

```diff
@@ -607,17 +608,46 @@ impl StructuredExtendedInformation {
         13 => deprecates_fpu_cs_ds,
         // 14 - reserved
         15 => pqe,
-        // 16-17 - reserved
+        16 => avx512f,
+        17 => avx512dq,
         18 => rdseed,
         19 => adx,
         20 => smap,
-        // 21-24 - reserved
-        25 => intel_processor_trace
-        // 26-31 - reserved
+        21 => avx512ifma,
```

Comment:
`avx512_ifma` to match the manual, please.

Reply:
Note that, for whatever reason, the [Intel intrinsics guide](https://software.intel.com/sites/landingpage/IntrinsicsGuide) calls these avx512ifma52...while [The Manual (October 2017)](https://software.intel.com/sites/default/files/managed/39/c5/325462-sdm-vol-1-2abcd-3abcd.pdf) as you mention calls these `AVX512_IFMA`.